### PR TITLE
Additional clarity for pylintrc_tweaks how-to.

### PR DESCRIPTION
**Pull request by benpatterson**
@nedbat @clintonb @cpennington

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -24,14 +24,23 @@ edx_lint master file::
...
 is detected, the edited file will be moved aside so it can be compared to the
 newly written file.
 
+Customizing edx_lint
+--------------------
+You can customize the resulting pylintrc file by creating a pylintrc_tweaks file in the
+current directory before running the ``write`` sub-command.  It should contain only the
+settings you want to override.
+
+**Note:** If your project is not a django project, you'll want to disable the django plugins with
```
Comment (clintonb) [inline README.rst]: Capitalize Django.

**Comment by clintonb** [comment]
I'm still not sure it makes sense to include the Django plugin(s) by default, since this isn't only intended for use with Django. However, we can hash that out later.

:+1:

File: README.rst (hunk 1 above)
Comment (benpatterson) [inline README.rst]: Done...amended commit & re-pushed.

**Comment by nedbat** [comment]
I agree that this is an unfortunate way to solve the problem.  I'll figure out a better way, but in the meantime, :+1:

**Pull request merged**